src/Response.php

```diff
@@ -7,12 +7,11 @@
 /**
  * Http response
  */
-class Response  implements ResponseInterface
+class Response implements ResponseInterface
 {
-    use Message\ProtocolVersion;
```

Comment:
`Response` should only use its own traits. Message traits should just serve as base.